File: src/main/java/com/now/nowbot/model/json/InfoLogStatistics.java
package com.now.nowbot.model.json;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.time.LocalDateTime;

public class InfoLogStatistics extends Statistics {
    @JsonProperty("time")
    LocalDateTime logTime;

    public LocalDateTime getLogTime() {
        return logTime;
    }

    public void setLogTime(LocalDateTime logTime) {
        this.logTime = logTime;
    }
}

File: src/main/java/com/now/nowbot/model/osu/InfoLogStatistics.kt
package com.now.nowbot.model.osu

import com.fasterxml.jackson.annotation.JsonProperty
import tools.jackson.databind.PropertyNamingStrategies
import tools.jackson.databind.annotation.JsonNaming
import java.time.LocalDateTime

@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
class InfoLogStatistics : Statistics() {
    @JsonProperty("time")
    var logTime: LocalDateTime = LocalDateTime.now()
}
